src/storage/index/LookUpEdgeIndexProcessor.cpp

```diff
@@ -42,8 +42,6 @@ void LookUpEdgeIndexProcessor::process(const cpp2::LookUpIndexRequest& req) {
             } else {
                 this->pushResultCode(this->to(code), partId);
             }
-            this->onFinished();
```

Comment:
We don't return now?

Reply:
Emm.. I don't think we need any return here. because this is last line of lambda block. Is there any other intention?

Reply:
> We don't return now?

let me  improve the logic,  let return directly when any part error occur.